doc/adr/0002-GraphQL.md

```diff
@@ -14,6 +14,36 @@ Date: 2022-03-15
 * We then [opened this issue][issue] in early January 2021.
 * [Alternatives to going from current REST to GraphQL][alt] seem limited; [more][more].
 
+#### Use cases yet to support
+
+These are [high level][org] (non-TSP specific), so will require focused [TSP][tsp] prototyping.
+
+1. ***Describe*** client schemas based on potentially refactored [domain][language] resolutions.
+1. ***Dimension*** client requests towards trace servers solely out of the needed information.
+1. Get potentially more ***predictable*** and scaled results with only the requested data ranges.
+1. ***Combine*** more than just one trace data source or server in request if resolvable so.
+1. Enable both API compatibility and ***evolution***, through being version-agnostic in requests.
+1. Allow potential ***custom*** resolution logic between clients and trace data sources, at scale.
+1. [Keep REST support or ***iterate*** towards GraphQL][iter] use in virtually any TSP client.
+
+#### And their stakeholders
+
+There is no mapping between these roles and aforementioned use cases, as many may apply to many.
+
+1. TSP ***client*** developers, users or integrators; [Python][client-py], [TypeScript][client-ts].
+1. Tracing ***application*** developers, either [Theia][theia] or Eclipse (or else, potentially).
```

Comment:
`Tracing` is the technique to collect data during program's execution. I think what you mean here is trace analysis and visualization, i.e. analysis and visualization of data collected during `Tracing`.